### PR TITLE
Documentation for 2.0 version

### PR DESCRIPTION
**Pull request by mzbroch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,23 +58,29 @@ The plugin behavior can be controlled with the following list of settings
...
-NetBox ahead of time and to provide them when you want to start the onboarding process.
+To properly onboard a device, the plugin needs to only know the Site as well as device's primary IP address or DNS Name.
 
 > For DNS Name Resolution to work, the instance of NetBox must be able to resolve the name of the
 > device to IP address.
 
+Providing other attributes (`Platform`, `Device Type`, `Device Role`) is optional - if any of these attributes is provided, plugin will use provided value for the onboarded device.
 If `Platform`, `Device Type` and/or `Device Role` are not provided, the plugin will try to identify these information automatically and, based on the settings, it can create them in NetBox as needed.
-> If the Platform is provided, it must contains a valid Napalm driver available to the worker in Python
+> If the Platform is provided, it must point the the existing NetBox Platform. NAPALM driver of this platform will be used only if it is defined for the platform in NetBox.
```
Comment (glennmatthews) [inline README.md:82]: "...it must point to an existing NetBox Platform." perhaps?

File: docs/release-notes/version-2.0.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+New attribute in the OnboardingDevice model allows for skipping the onboarding request on devices with disabled onboarding setting.
+
+#### Improved logging
+
+Logging has been improved and refactored.
+
+#### Onboarding process refactor
+
+Onboarding process has been completely refactored, including classes disaggregation.
```
Comment (glennmatthews) [inline docs/release-notes/version-2.0.md:37]: All these separate sections make this pretty verbose - personally I'd rather follow the [NetBox release note format](https://netbox.readthedocs.io/en/stable/release-notes/version-2.9/) and just have bulleted lists under "Enhancements" and "Bug Fixes" except where a major change requires a more in-depth (paragraph-length) description.

File: README.md (hunk 1 above)
Comment (mzbroch) [reply README.md:82]: the the the You are correct :-)

File: docs/release-notes/version-2.0.md (hunk 2 above)
Comment (mzbroch) [reply docs/release-notes/version-2.0.md]: Did a shorter version

**Pull request merged**